src/interfaces/wallet.h

```diff
@@ -326,7 +327,7 @@ class WalletLoader : public ChainClient
    virtual std::string getWalletDir() = 0;
 
    //! Restore backup wallet
-   virtual std::unique_ptr<Wallet> restoreWallet(const std::string& backup_file, const std::string& wallet_name, bilingual_str& error, std::vector<bilingual_str>& warnings) = 0;
+   virtual std::unique_ptr<Wallet> restoreWallet(const fs::path& backup_file, const std::string& wallet_name, bilingual_str& error, std::vector<bilingual_str>& warnings) = 0;
```

Comment:
In commit "Change type of `backup_file` parameter in RestoreWallet/restoreWallet" (3a45dc36a663ea67f13e7d5c413518b9385ec594)

Hmm, interesting that this method doesn't seem to be called anywhere. I guess whatever PR which will use it will also need to be updated.

Reply:
https://github.com/bitcoin-core/gui/pull/471 is the PR using it.